pages/1.12/deploying-services/private-docker-registry/index.md

```diff
@@ -235,8 +235,45 @@ Follow these steps to add your Docker registry credentials to the [DC/OS Enterpr
 
    The Docker image will now pull using the provided security credentials.
 
+<a name="docker-repo-certs"></a>
+
+# Configuring the Docker registry to use a custom certificate
+As an alternative to providing user credentials to authorize access to the private Docker registry, you can use a custom certificate to authorize access to the private Docker registry.
+
+To configure a custom certificate for accessing the private Docker registry and DC/OS UCR:
+
+1. Create or identify a custom certificate that you want to use as a trusted certificate for accessing the Docker registry.
```

Comment:
I'd add, "And then configure the registry to use this certificate, following the instructions provided by the registry provider".

Reply:
This seems like it would go after the step.